packages/api/src/hooks/p2p/entity/advert/p2p-advertiser-adverts/useAdvertiserAdverts.ts

```diff
@@ -13,11 +13,12 @@ const useAdvertiserAdverts = (
         options: {
             ...config,
             getNextPageParam: (lastPage, pages) => {
-                if (!lastPage?.p2p_advertiser_adverts?.list) return;
+                if (!lastPage?.p2p_advertiser_adverts?.list?.length) return;
 
                 return pages.length;
             },
             enabled: isSuccess && (config?.enabled === undefined || config.enabled),
+            refetchOnWindowFocus: false,
```

Comment:
@nada-deriv, the ads don't change that frequent right? If they do, I'd suggest leaving `refetchOnWindowFocus` set to `true`
```suggestion
            refetchOnWindowFocus: false,
```